index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>My Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+    <div class="logo">my logo</div>
+    <div class="menu">
+      <img src="./images/Menu.png" alt="menu">
+    </div>
+  </header>
```

Comment:
- [ ] Kindly reduce the spacing at the top of the navbar section to match the Figma design.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/220366052-f1960532-b053-47eb-962f-05d8843d6d22.png) | ![image](https://user-images.githubusercontent.com/62290545/220366367-8c739323-9630-4e1a-867c-c5f998ba5b8c.png) |

Reply:
Thanks for your kind review!